core-libs/setup/ssr/logger/loggers/default-express-server-logger.spec.ts

```diff
@@ -1,7 +1,22 @@
+import * as isDevModeFn from '@angular/core';
```

Comment:
[NITPICK]

`*` is not devModeFn, but the whole `@angular/core` library contents
```suggestion
import * as angularCore from '@angular/core';
```